src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/project.json

```diff
@@ -45,7 +45,6 @@
         },
         "System.Runtime.Serialization.Primitives": "4.3.0-*"
       }
-    },
-    "net451": {}
```

Comment:
Tried it out manually. The CLI invokes the tool correctly for `net451` only and `netcoreapp1.x` only projects. Not sure if we'd want to add a functional test to make sure it runs in these cases.